code/_onclick/hud/movable_screen_objects.dm

```diff
@@ -8,4 +8,4 @@
 		var/list/screen_loc_params = splittext(PM["screen-loc"], ",")
 		var/list/x_data = splittext(screen_loc_params[1], ":")
 		var/list/y_data = splittext(screen_loc_params[2], ":")
-		screen_loc = "LEFT+[x_data[1]]:[text2num(x_data[2])-16],BOTTOM+[y_data[1]]:[text2num(y_data[2])-16]"
+		screen_loc = "LEFT+[x_data[1]]:[text2num(x_data[2])-(16 + world.icon_size)],BOTTOM+[y_data[1]]:[text2num(y_data[2])-(16 + world.icon_size)]"
```

Comment:
Are you sure this is the right math? If it's really respecting `world.icon_size`, why is there still a hard-coded `16`?